docs/data-cookbooks/browse-generation.md

```diff
@@ -78,173 +76,205 @@ For this example, you are going to be adding two workflows to your Cumulus deplo
 
 #### Workflow Configuration
 
-Add the following to a new file ```browseExample.yml``` in your deployment's main directory (the same location your app directory, lambdas.yml, etc are), copy the example file [from github](https://github.com/nasa/cumulus/blob/master/example/workflows/browseExample.yml).  The file should contain the two example workflows.
+Add the following to a new file ```browse_example.tf``` in your deployment's main directory: [from github](https://github.com/nasa/cumulus/blob/master/example/workflows/browseExample.yml).  The file should contain the two example workflow modules.
+
+**Please Note**: You should update the `source =` line to match the current Cumulus `workflow` module deployment artifact to the version of Cumulus you're deploying:
+
+```tcl
+source = "https://github.com/nasa/cumulus/releases/download/{version}/terraform-aws-cumulus.zip//tf-modules/cumulus"
+```
 
 A few things to note about tasks in the workflow being added:
 
 * The CMR step in CookbookBrowseExample:
 
-```yaml
-  CmrStep:
-    Parameters:
-      cma:
-        event: '$'
-        task_config:
-          bucket: '{$.meta.buckets.internal.name}'
-          stack: '{$.meta.stack}'
-          cmr: '{$.meta.cmr}'
-          process: '{$.meta.process}'
-          input_granules: '{$.meta.input_granules}'
-          granuleIdExtraction: '{$.meta.collection.granuleIdExtraction}'
-    Type: Task
-    Resource: ${PostToCmrLambdaFunction.Arn}
-    Catch:
-      - ErrorEquals:
-        - States.ALL
-        ResultPath: '$.exception'
-        Next: WorkflowFailed
-    End: true
+```json
+"CmrStep": {
+      "Parameters": {
```

Comment:
Can we fix the indentation in this JSON block? Makes it easier to copy/paste

Reply:
Sure can: 21900971d